package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "server",
+  "version": "0.0.0",
+  "private": true,
+  "scripts": {
+    "start": "node ./server/bin/www",
+    "dev": "nodemon ./server/bin/www",
+    "dev:both": "cd client && npm start && cd .. && nodemon ./server/bin/www",
```

Comment:
I think we forgot to remove this line, it's not working

Reply:
It works for me. What's wrong with this line?

I'd maybe suggest using concurrently to run both processes (and kill both if one crashes)

Reply:
Concurrently has now been added and the script updated.